src/SFA.DAS.EmployerAccounts.Web/Web.config

```diff
@@ -19,7 +19,8 @@
     <add key="TokenCertificateThumbprint" value="6B7ACC520305BFDB4F7252DAEB2177CC091FAAE1"/>
     <add key="UnobtrusiveJavaScriptEnabled" value="false"/>
     <add key="webpages:Enabled" value="false"/>
-    <add key="webpages:Version" value="3.0.0.0"/>    
+    <add key="webpages:Version" value="3.0.0.0"/>
+    <add key="SupportConsoleUser" value="Tier1User,Tier2User"/>
```

Comment:
This config value needs to be read from table storage in the EmployerAccountConfiguration